docs/reference/sql/endpoints/client-apps/tableau-server.asciidoc

```diff
@@ -0,0 +1,54 @@
+[role="xpack"]
+[testenv="platinum"]
+[[sql-client-apps-tableau-server]]
+=== Tableau Server
+
+Use the {es} JDBC driver and dedicated {es} Tableau Connector to access {es} data from Tableau Server.
+
+IMPORTANT: Elastic does not endorse, promote or provide support for this application; for native Elasticsearch integration in this
+product, please reach out to its vendor.
+
+==== Prerequisites
+
+* https://www.tableau.com/products/server[Tableau Server] 2019.4 or later
+* {es-sql} <<sql-jdbc, JDBC driver>>
+* https://www.elastic.co/downloads/tableau-connector[{es} Connector for Tableau]
+
+==== Data loading
+
+First place the downloaded JDBC driver in Tableau Server drivers directory:
+
+* Windows: `C:\Program Files\Tableau\Drivers`
+* Mac: `/Users/[user]/Library/Tableau/Drivers`
+
+and the {es} Connector for Tableau in Tableau Server connectors directory. Refer to Tableau Server documentation or make use of the TSM
+command line interface to find out the location of this directory for your installation.
+
+Then (re)start Tableau Server.
```

Comment:
```suggestion
Start Tableau Server.
```

Reply:
Updated and committed the suggestion: the Server is very likely running by the time a user would configure the connector, unlike the Desktop.